src/components/index-page/sections/organizers.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+import Section from "../../base-section/base-section"
+import "./organizers.css"
+
+const Organizers = () => {
+  return (
+    <Section classes="about" id="DevFest">
```

Comment:
Hey @fakham, can you change the **class name** `about` to `organizers`
and **id** `Devfest` to `Organizers`

Reply:
Done